oxidation/libparsec/crates/core/src/logged_core.rs

```diff
@@ -0,0 +1,60 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 (eventually AGPL-3.0) 2016-present Scille SAS
+
+use futures::lock::Mutex;
+use once_cell::sync::OnceCell;
+
+use libparsec_types::DeviceID;
+
+use crate::{LoggedCoreError, LoggedCoreResult};
+
+static CORE: OnceCell<Mutex<Vec<Option<LoggedCore>>>> = OnceCell::new();
+
+#[derive(Debug, Clone)]
+struct LoggedCore {
+    // TODO: Remove me when LoggedCore is implemented
+    test_device_id: DeviceID,
+}
+
+/// Output: i32 is because javascript number is a 64 bits float
+pub async fn login(test_device_id: DeviceID) -> i32 {
```

Comment:
Returning a naked `i32` is bad for typing.
use a struct wrapping the i32 with from/to conversion instead.

Reply:
Done